app/main.ipynb

```diff
@@ -92,16 +188,33 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "df[\"item_price\"] = df[\"orderitem_quantity\"] * df[\"product_price\"]\n",
+    "top10_by_price = df.groupby(\"product_name\").sum()[\"item_price\"].nlargest(10)\n",
+    "top10_by_price_pie = top10_by_price.plot(kind=\"pie\", autopct='%1.1f%%')\n",
+    "plt.title(\"Top 10 Products by Item Price\")\n",
+    "plt.show()"
```

Comment:
Make diagram bigger

Reply:
all comments are taken into account and corrected